SearchInRotatedArray.java

```diff
@@ -0,0 +1,38 @@
+class Solution {
+    public int search(int[] nums, int target) {
```

Comment:
This code is not accepted on leetcode. Please try to solve it or reach out to me, the TA or the professor.